### ISSUE TITLE
Type errors

### ISSUE THREAD

**Issue by rienswagerman** [nozer/quill-delta-to-html#28]
Hi. I get the following type errors:

```
 ERROR in /home/circleci/project/node_modules/quill-delta-to-html/dist/commonjs/mentions/MentionSanitizer.d.ts
    ERROR in /home/circleci/project/node_modules/quill-delta-to-html/dist/commonjs/mentions/MentionSanitizer.d.ts(4,5):
    TS2411: Property ''name'' of type 'string | undefined' is not assignable to string index type 'string'.
    
    ERROR in /home/circleci/project/node_modules/quill-delta-to-html/dist/commonjs/mentions/MentionSanitizer.d.ts
    ERROR in /home/circleci/project/node_modules/quill-delta-to-html/dist/commonjs/mentions/MentionSanitizer.d.ts(5,5):
    TS2411: Property ''target'' of type 'string | undefined' is not assignable to string index type 'string'.
    
    ERROR in /home/circleci/project/node_modules/quill-delta-to-html/dist/commonjs/mentions/MentionSanitizer.d.ts
    ERROR in /home/circleci/project/node_modules/quill-delta-to-html/dist/commonjs/mentions/MentionSanitizer.d.ts(6,5):
    TS2411: Property ''slug'' of type 'string | undefined' is not assignable to string index type 'string'.
    
    ERROR in /home/circleci/project/node_modules/quill-delta-to-html/dist/commonjs/mentions/MentionSanitizer.d.ts
    ERROR in /home/circleci/project/node_modules/quill-delta-to-html/dist/commonjs/mentions/MentionSanitizer.d.ts(7,5):
    TS2411: Property ''class'' of type 'string | undefined' is not assignable to string index type 'string'.
    
    ERROR in /home/circleci/project/node_modules/quill-delta-to-html/dist/commonjs/mentions/MentionSanitizer.d.ts
    ERROR in /home/circleci/project/node_modules/quill-delta-to-html/dist/commonjs/mentions/MentionSanitizer.d.ts(8,5):
    TS2411: Property ''avatar'' of type 'string | undefined' is not assignable to string index type 'string'.
    
    ERROR in /home/circleci/project/node_modules/quill-delta-to-html/dist/commonjs/mentions/MentionSanitizer.d.ts
    ERROR in /home/circleci/project/node_modules/quill-delta-to-html/dist/commonjs/mentions/MentionSanitizer.d.ts(9,5):
    TS2411: Property ''id'' of type 'string | undefined' is not assignable to string index type 'string'.
    
    ERROR in /home/circleci/project/node_modules/quill-delta-to-html/dist/commonjs/mentions/MentionSanitizer.d.ts
    ERROR in /home/circleci/project/node_modules/quill-delta-to-html/dist/commonjs/mentions/MentionSanitizer.d.ts(10,5):
    TS2411: Property ''end-point'' of type 'string | undefined' is not assignable to string index type 'string'.
```

tsconfig:

```
{
  "compilerOptions": {
    "module": "esnext",
    "moduleResolution": "node",
    "target": "es5",
    "lib": [
      "dom",
      "es6",
      "dom.iterable",
      "scripthost",
      "es2017"
    ],
    "sourceMap": true,
    "emitDecoratorMetadata": true,
    "experimentalDecorators": true,
    "noImplicitAny": true,
    "noImplicitThis": true,
    "suppressImplicitAnyIndexErrors": true,
    "noUnusedParameters": true,
    "noUnusedLocals": true,
    "strictNullChecks": true,
    "alwaysStrict": true,
    "allowSyntheticDefaultImports": true,
    "allowJs": true,
    "noFallthroughCasesInSwitch": true
  },
  "include": [
    "./assets/src/**/*"
  ],
  "exclude": [
    "node_modules"
  ]
}

```

Thanks!

**Comment by nozer** [comment]
What is the typescript version you are using?

**Comment by rienswagerman** [comment]
Tried both 2.9.1 and 3.0. Currently on 3.0

**Comment by nozer** [comment]
Hi, it was due to you having these settings in tsconfig:
```
"noImplicitThis": true,
"strictNullChecks": true,
```

I changed my tsconfig settings to these and fixed the typescript errors and pushed as `v0.9.2`. 
Thanks

**Comment by nozer** [comment]
Aaargh, you have other settings that also give typescript errors. I will fix and push again. Hang in there! :)

**Comment by nozer** [comment]
Fixed and pushed as `v0.9.3`

**Comment by rienswagerman** [comment]
Perfect! Thanks for the quick fixes!
